anyway/config.py

```diff
@@ -12,7 +12,7 @@
 SQLALCHEMY_TRACK_MODIFICATIONS = True
 ENTRIES_PER_PAGE = os.environ.get("ENTRIES_PER_PAGE", 1000)
 SQLALCHEMY_ENGINE_OPTIONS = {}
-# SQLALCHEMY_ECHO = True
+SQLALCHEMY_ECHO = True
```

Comment:
Needed?

Reply:
No... I use it for debugging and forgot to remove.